app/views/home/home.html.erb

```diff
@@ -126,6 +125,6 @@
 </style>
 
 <% cache('feature_home-footer', skip_digest: true) do %>
-  <%= feature("home-footer") %>
```

Comment:
Hi, was this done incorrectly somehow? It might just be a feature we're not currently using. I think footer-notice is the topics table, so it might display twice if we add it here? But I could be wrong.

Reply:
Can you confirm this? Then I think we are ready to merge, thanks!

Reply:
Hey! So in the older ```home.html.erb```, this is there so I haven't changed it yet. Would you like me to remove this?

Reference:

https://github.com/publiclab/plots2/blob/5aa872006c56507b78b3641952a853f01e882b72/app/views/home/home.html.erb#L128-L130